admin-dev/themes/new-theme/scss/pages/_translation_page.scss

```
@@ -1,5 +1,162 @@
+$color-error: #c05c67 !default;
+$color-active: #25b9d7 !default;
```

```suggestion
$translations-error: #c05c67 !default;
$translations-active: #25b9d7 !default;
```

I think you should set a more specific name, in case something use the same variable without setting it as default, it will be override and your work won't be ISO

---

```suggestion
$color-error: #c05c67 !default;
```

Also you can remove the active, as it's `$primary`